.github/workflows/ci.yml

```diff
@@ -10,15 +10,11 @@ jobs:
         working-directory: ui
     steps:
       - uses: actions/checkout@v2
-      - run: docker build .
-      # We run the build a second time instead of trying to wire the docker SHA from 
-      # one step to another. The layers should be cached, so it should be really fast.
-      # We run the build in a separate step above so that there's nice output if something
-      # doesn't work.
-      - run: docker run --name ui --rm -d -p 4000:4000 $(docker build -q .)
-      # We don't have UI tests yet, so we just make sure the `/` route returns 200.
-      - run: curl --fail http://localhost:4000
-      - run: docker stop ui
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.21.0
+      - run: npm install
+      - run: npm test
```

Comment:
@codeviking do you think it's worth keeping that "see if it builds" test?

Reply:
I do, for two reasons:

1. I think some folks work on the code without using a container -- which means they might inadvertently break `docker build`.
2. If it can't build the deploy will fail (it has to run `docker build`) as part of that process. So making sure things build before merging them helps the commit jitter that occurs when someone fixes forward.